### DISCUSSION TITLE
Math, prompts, extendend Markdown support, and something else

### DISCUSSION THREAD

**Discussion by fabriziobagala** [forem/forem#19443] [Feature Requests]
Using the editor, I noticed some missing features that I would like to see integrated:

- Support math via MathJax (https://www.mathjax.org/)
- Support for prompts (like the example below):
![Prompts](https://user-images.githubusercontent.com/62190484/236910449-9363b38d-1355-4107-952f-ca462b2d9061.png)
- Support for extended Markdown (https://www.markdownguide.org/extended-syntax/)
- Support for Font Awesome (https://fontawesome.com/)

Moreover, in the profile, I would like:

- To be able to link my social pages (Facebook, LinkedIn, Mastodon, Twitter, GitHub, Bluesky) without needing to link. The functionality should be similar to that introduced in the GitHub profile (https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#adding-links-to-your-social-accounts)
- To have a blue mark next to the name to identify a trusted user.

Lastly, on the account page, ability to enter two-factor authentication.

**Comment by ker2x** [comment]
Almost 3 years and still no Mathjax ?
